src/main/java/org/interledger/btp/BilateralTransferProtocolResponse.java

```diff
@@ -0,0 +1,129 @@
+package org.interledger.btp;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public interface BilateralTransferProtocolResponse extends BilateralTransferProtocolPacket {
+
+  /**
+   * Get the default builder.
+   *
+   * @return a {@link Builder} instance.
+   */
+  static Builder builder() {
+    return new Builder();
+  }
+
+
+  /**
+   * A builder for instances of {@link BilateralTransferProtocolResponse}.
+   */
+  class Builder {
+
+    private long requestId;
```

Comment:
So, I think we might consider separating the `Packet` data from the payload. As an example, in the ASN the object called `BilateralTransferProtocolPacket` has a `requestId`, but the actual `Response` object has only subprotocol data.

In my implementation, I'm not going to be using `BilateralTransferProtocolPacket` because I won't be running on a Websocket, and thus won't need to track request/response identifiers for correlation. However, I _will_ want to re-use the Prepare/Fulfill/Reject/Message/Error interfaces. If we don't have a good separation between the `Packet` (i.e., encoded ASN.1) and the business objects, I won't be able to do that.